scripts/ci/coverage.sh

```diff
@@ -1,29 +1,23 @@
 #!/bin/bash
 set -e
 
-# CC LD AR CXX has to be predifubed
-#
-
-export PKG_CONFIG_PATH="$HOME/cunit-install/x86_64/lib/pkgconfig:${PKG_CONFIG_PATH}"
-
-CWD=$(dirname "$0")
-TDIR=`mktemp -d -p ~`
-
-cd ${TDIR}
-echo 1000 | tee /proc/sys/vm/nr_hugepages
-mkdir -p /mnt/huge
-mount -t hugetlbfs nodev /mnt/huge
+if [ "${CC#clang}" != "${CC}" ] ; then
+	export CXX="clang++"
+fi
 
-git clone ${CWD}/../../ odp
-cd ./odp
+cd "$(dirname "$0")"/../..
```

Comment:
Because I want to be able to retrieve logs after build fails. With your scripts `set -e` ensures that build script exits as soon as anything fails, leaving us with empty `after_failure` output.

Reply:
gh... yep. Maybe  how about trap function just to print logs to stdout?
function print_logs()
{
}
trap print_logs

Reply:
I don't like that. This might easily leave us with no logs in case of any bash error. And having build in a random-name dir leaves us with no easy way to recover them. You can always do git clone locally and run script using new copy.

Reply:
in case if any bash error 1) bash error will be visible on stdout,stderr 2) this function will called and you can do something with logs. Linaro Jenkins for example calls script https://git.linaro.org/ci/publishing-api.git/plain/linaro-cp.py  to upload artifacts to snapshot.linaro.org. But now I do not see that other projects do upload logs. We can just find all logs and print them and Travis will store them. Or else upload them somewhere.

Reply:
@muvarov For now I'm trying to restore back functionality that existed before merging container-based config. You can submit trap-error-logs-etc later.

Reply:
yes, but for now can you leave git clone, because it's already exist in the code. Removing it and pushing back is a little bit wrong workflow.

Reply:
@muvarov well, leaving us without error logs was quite a wrong workflow.

Reply:
In other words: `scripts/ci/*.sh` should first serve needs of CI integration rather that one's personal needs wrt his containers (e.g. I'd like to do a out-of-tree build in host-determined directory, because that better fits my development process, but it is not necessary and a bit counterproductive for CI builds).